src/actions.py

```diff
@@ -160,12 +171,13 @@ def get_deposite_amount(self, bank_acc_no):
 					}
 				)
 			print(res)
-			res.append({"Sum of Deposits": sum })
+			res.append({"Sum of Deposit": sum })
 			return res
 		except Exception as e:
 			print(e)
 			return {}
 
+
 	def delete_user(self, bank_acc_no):
 		try:
 			items = self.item_repo.delete_user(bank_acc_no)
```

Comment:
variable name can be improved. The `delete_user` function will return a deleted user.